openmetadata-ui/src/main/resources/ui/src/pages/DataInsightPage/DataInsightPage.component.tsx

```diff
@@ -77,14 +80,27 @@ const DataInsightPage = () => {
   const { isAdminUser } = useAuth();
   const history = useHistory();
 
-  const [teamsOptions, setTeamOptions] = useState<SelectProps['options']>([]);
+  const [teamsOptions, setTeamOptions] = useState<TeamStateType>({
+    defaultOptions: [],
+    selectedOptions: [],
+    options: [],
+  });
```

Comment:
Create an initial filter object and use it here.

Reply:
I think it is fine for such simple data